src/Auth/EmailVerification/EmailVerificationController.php

```diff
@@ -0,0 +1,108 @@
+<?php
+namespace Nodes\Api\Auth\EmailVerification;
+
+use Illuminate\Routing\Controller as IlluminateController;
+use Illuminate\Support\Facades\Input;
+use Nodes\Api\Auth\Exceptions\MissingUserModelException;
+use Nodes\Api\Routing\Helpers as ApiHelpers;
+use Nodes\Exceptions\Exception;
+
+/**
+ * Class EmailVerificationController
+ *
+ * @package Nodes\Api\Auth\EmailVerification
+ */
+class EmailVerificationController extends IlluminateController
```

Comment:
This is maybe a bit nazi, but we usually also name Controllers in plural. So in this case it would be EmailVerification**s**